pygeoapi/plugin.py

```diff
@@ -46,7 +46,10 @@
         'MongoDB': 'pygeoapi.provider.mongo.MongoProvider',
         'FileSystem': 'pygeoapi.provider.filesystem.FileSystemProvider',
         'rasterio': 'pygeoapi.provider.rasterio_.RasterioProvider',
-        'xarray': 'pygeoapi.provider.xarray_.XarrayProvider'
+        'xarray': 'pygeoapi.provider.xarray_.XarrayProvider',
+        'tiles': {
```

Comment:
The idea is that `tiles` is referring to the spec that could have its own set of providers. In fact, `mvt` doesn't inherit from the standard base class but it has its own https://github.com/geopython/pygeoapi/pull/419/files#diff-69ec7115fb0a64f88804716b46ea7de5R37 which is specific for the tiles extension. One of the advantages is to avoid a potential conflict of names with other providers. For instance, `gdal/ogr` could act as a provider for different specifications even for MVT.

Reply:
should we put `tiles` then at the same level of providers, since it's not 'provider' based ?

Reply:
What do you think of this change?

```python
'provider': {...},
'tiles': {
        'provider': {
            'MVTProvider': 'pygeoapi.provider.mvt.MVTProvider'
        }
},
```